erroneous_throw_promise.m:009: Error: [38;5;87m`foo'(in)[39;49m has determinism erroneous but
erroneous_throw_promise.m:009:   also has foreign clauses that have a
erroneous_throw_promise.m:009:   `will_not_throw_exception' attribute. [38;5;203mThis[39;49m
erroneous_throw_promise.m:009:   [38;5;203mattribute cannot be applied to erroneous[39;49m
erroneous_throw_promise.m:009:   [38;5;203mprocedures.[39;49m
erroneous_throw_promise.m:011: Error: [38;5;87m`bar'(in)[39;49m has determinism erroneous but
erroneous_throw_promise.m:011:   also has foreign clauses that have a
erroneous_throw_promise.m:011:   `will_not_throw_exception' attribute. [38;5;203mThis[39;49m
erroneous_throw_promise.m:011:   [38;5;203mattribute cannot be applied to erroneous[39;49m
erroneous_throw_promise.m:011:   [38;5;203mprocedures.[39;49m
